src/app/pages/storage/disks/disk-list/disk-list.component.ts

```diff
@@ -223,6 +233,10 @@ export class DiskListComponent implements EntityTableConfig<Disk> {
         entityList.getData();
       }
     });
+    this.slideInService.onClose$.pipe(untilDestroyed(this)).subscribe(() => {
+      entityList.getData();
+      entityList.pageChanged();
```

Comment:
Is `pageChanged` really needed?

Reply:
Yes, it is necessary to uncheck all checkboxes

https://user-images.githubusercontent.com/22006748/167824868-2a4895cc-eb50-4630-b046-4a81b23ff840.mp4

](url)